src/gfx/gfxutils.js

```diff
@@ -196,6 +196,25 @@ THREE.Raycaster.prototype.intersectVisibleObject = function (gfxObj, camera, cli
   return p;
 };
 
+THREE.Matrix4.prototype.extractScale = (function () {
```

Comment:
It's great to have such a function. Is there any other place in code to use it?

Reply:
I checked, there is no other place in which we can use it